ipaserver/install/bindinstance.py

```diff
@@ -423,17 +423,15 @@ def zonemgr_callback(option, opt_str, value, parser):
             encoding = getattr(sys.stdin, 'encoding', None)
             if encoding is None:
                 encoding = 'utf-8'
-            value = value.decode(encoding)
+            if not isinstance(value, unicode):
```

Comment:
> OptionParser behaves differently in Python 2/3 so it actually makes sense to do this

Yep :)
Good point, I'll change the commit message and add a code comment there as well. What do you think it would be a good comment that express what you're saying? 

Something like this? 
```python
...
# value is str in py2 and py3
# in py3 isinstance(value, unicode) returns True
if not isinstance(value, unicode):
...
```